src/test/java/core/basesyntax/services/impl/FileReaderServiceImplTest.java

```diff
@@ -0,0 +1,49 @@
+package core.basesyntax.services.impl;
+
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class FileReaderServiceImplTest {
+    private static final String PATH_EMPTY_FILE =
+            "src/test/java/core/basesyntax/resources/input_transactions_empty.csv";
+    private static final String PATH_TRANSACTIONS =
+            "src/test/java/core/basesyntax/resources/input_transactions.csv";
+    private static FileReaderServiceImpl fileReaderService;
+
+    @BeforeClass
+    public static void beforeClass() {
+        fileReaderService = new FileReaderServiceImpl();
+    }
+
+    @Test (expected = RuntimeException.class)
+    public void fileReader_incorrectFile_notOk() {
```

Comment:
```suggestion
    public void read_nonExistentFilePath_notOk() {
```

There are a lot of ways to name your test methods. The main point is that they should have informative names and be consistent along with other developers in your team. For this task use such convention: `<methodUnderTest>_<state>_<expectedBehavior>`; For example, if we are testing the method register with a `null` user's age the test method name should be `register_nullAge_notOk`. `notOk` is because the test expects the register method to throw an exception.